rskj-core/src/test/java/co/rsk/peg/BridgeUtilsTest.java

```diff
@@ -2914,4 +2919,327 @@ private Federation getGenesisFederationForTest(BridgeConstants bridgeConstants,
 
         return federation;
     }
+
+    @Test
+    public void getMinimumPegInTxValue_before_RSKIP219() {
+        when(activations.isActive(ConsensusRule.RSKIP219)).thenReturn(false);
+
+        Coin minimumPeginTxValue = bridgeConstantsRegtest.getLegacyMinimumPeginTxValueInSatoshis();
+        assertEquals(
+            minimumPeginTxValue,
+            BridgeUtils.getMinimumPegInTxValue(activations, bridgeConstantsRegtest)
+        );
+    }
+
+    @Test
+    public void getMinimumPegInTxValue_after_RSKIP219() {
+        when(activations.isActive(ConsensusRule.RSKIP219)).thenReturn(true);
+
+        Coin minimumPeginTxValue = bridgeConstantsRegtest.getMinimumPeginTxValueInSatoshis();
+        assertEquals(
+            minimumPeginTxValue,
+            BridgeUtils.getMinimumPegInTxValue(activations, bridgeConstantsRegtest)
+        );
+    }
+
+    @Test
+    public void testIsAnyUTXOAmountBelowMinimum_has_utxos_below_minimum() {
+        Coin minimumPegInTxValue = BridgeUtils.getMinimumPegInTxValue(activations, bridgeConstantsRegtest);
+        Coin valueBelowMinimum = minimumPegInTxValue.minus(Coin.SATOSHI);
+        Coin valueAboveMinimum = minimumPegInTxValue.plus(Coin.SATOSHI);
+
+        BtcTransaction btcTx = new BtcTransaction(bridgeConstantsRegtest.getBtcParams());
+        btcTx.addInput(Sha256Hash.ZERO_HASH, 0, new Script(new byte[]{}));
+
+        List<Address> addresses = Arrays.asList(
+            PegTestUtils.createRandomBtcAddress(bridgeConstantsRegtest.getBtcParams()),
+            PegTestUtils.createRandomBtcAddress(bridgeConstantsRegtest.getBtcParams()),
+            PegTestUtils.createRandomBtcAddress(bridgeConstantsRegtest.getBtcParams())
+        );
+
+        btcTx.addOutput(valueBelowMinimum, addresses.get(0));
+        btcTx.addOutput(minimumPegInTxValue, addresses.get(1));
+        btcTx.addOutput(valueBelowMinimum, addresses.get(2));
+        btcTx.addOutput(valueAboveMinimum, addresses.get(2));
+
+        assertTrue(
```

Comment:
You could add a couple of assertions here

- Calling with only address[0]
- Calling with only address[2]
- Perhaps also calling with addresses 0 and 1, 0 and 2
- Only address 1 (should be false)

Reply:
Done